addons/stock/models/stock_picking.py

```diff
@@ -140,6 +141,20 @@ class PickingType(models.Model):
     show_picking_type = fields.Boolean(compute='_compute_show_picking_type')
 
     picking_properties_definition = fields.PropertiesDefinition("Picking Properties")
+    description = fields.Char(
+        'Overview Description', copy=True,
+        compute='_compute_description', store=True, precompute=True, readonly=False,
+    )
+    # By default, each picking type is in favorites for every user.
+    # It means that the relation has to store users who decided to remove a
+    # picking type from their favorites.
+    not_favorite_user_ids = fields.Many2many(
+        'res.users', 'picking_type_not_favorite_user_rel', 'picking_type_id', 'user_id',
+    )
+    is_favorite = fields.Boolean(
+        compute='_compute_is_favorite', inverse='_inverse_is_favorite', search='_search_is_favorite',
+        compute_sudo=True, string='Show Operation in Overview'
+    )
```

Comment:
I'm not a big fan of that. We could do the same as project. We just don't need to enable the field by default and let the user do it

Reply:
And hiding the star in the picking type setting is strange too. I would just do as project

Reply:
As discussed: the flag is no longer enabled by default, and the default filter was removed. The star widget is still hidden. Additionally, picking types are now ordered by `is_favorite` first.